plugins/providers/docker/driver.rb

```diff
@@ -151,6 +151,17 @@ def docker_bridge_ip
         end
       end
 
+      def exec(params, **opts, &block)
```

Comment:
The docker provider makes heavy use of **opts, I'm not sure how to modify just my changes to not use **opts without refactoring the the rest of the plugin